server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/event/ComputeMessagePreviewListenerTest.java

```diff
@@ -85,9 +84,8 @@ void setup() throws Exception {
 
         FakeAuthenticator authenticator = new FakeAuthenticator();
         authenticator.addUser(BOB, "12345");
-        authenticator.addUser(ADMIN, "admin123");
 
-        SessionProvider sessionProvider = new SessionProvider(authenticator, FakeAuthorizator.forUserAndAdmin(ADMIN, BOB));
+        SessionProvider sessionProvider = new SessionProvider(authenticator, FakeAuthorizator.forUserAndAdmin(BOB, BOB));
```

Comment:
Is this change really for the best as the first parameter of `FakeAuthorizator.forUserAndAdmin` should be an admin ?

Reply:
@chibenwa seemed to say it was not really necessary in a comment above to have `admin`... but I can put it back if necessary

Reply:
Use `FakeAuthaurizator.defaultReject()`